setup.py

```diff
@@ -79,6 +79,51 @@ def localversion_func(version: ScmVersion) -> str:
     return get_local_node_and_date(version)
 
 
+_deps = [
```

Comment:
since we're no longer dynamically appending to deps / dev_deps, i think having these directly in `setup(...` is more readable, what do people think?